### ISSUE TITLE
(AWS::ElasticLoadBalancingV2::Listener) false-positive E3008 for `LoadBalancerArn`

### ISSUE THREAD

**Issue by r-heimann** [aws-cloudformation/cfn-lint#2934]
### CloudFormation Lint Version

v0.83.1

### What operating system are you using?

Windows 10

### Describe the bug

`AWS::ElasticLoadBalancingV2::LoadBalancer` now support `!GetAtt <ResourceName>.LoadBalancerArn`, but `AWS::ElasticLoadBalancingV2::Listener` display this error message:

```
[cfn-lint] E3008: Property "LoadBalancerArn" has no valid Fn::GetAtt options at
Resources/<ResourceName>/Properties/LoadBalancerArn/Fn::GetAtt
```

### Expected behavior

`!GetAtt <ResourceName>.LoadBalancerArn` works.

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09

Resources:
  NLB:
    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
    Properties:
      IpAddressType: ipv4
      Scheme: internal
      Type: network
      SubnetMappings:
        - SubnetId: <Subnet>

  Listener:
    Type: AWS::ElasticLoadBalancingV2::Listener
    Properties:
      LoadBalancerArn: !GetAtt NLB.LoadBalancerArn
      Port: 8443
      Protocol: TCP
      DefaultActions:
        - Type: forward
          TargetGroupArn: !GetAtt TargetGroup.TargetGroupArn

  TargetGroup:
    Type: AWS::ElasticLoadBalancingV2::TargetGroup
    Properties:
      VpcId: <VPC>
      IpAddressType: ipv4
      Port: 8443
      Protocol: TCP
      TargetType: ip
      TargetGroupAttributes:
        - Key: preserve_client_ip.enabled
          Value: true
        - Key: deregistration_delay.connection_termination.enabled
          Value: true
      HealthCheckEnabled: true
      HealthCheckPort: traffic-port
      HealthCheckProtocol: TCP
      HealthyThresholdCount: 5
      UnhealthyThresholdCount: 5
```
